onnxruntime/core/protobuf/onnx.in.proto

```diff
@@ -535,6 +554,8 @@ message TypeProto {
 
     SparseTensor sparse_tensor_type = 8;
 
+    TimePoint    timepoint_type = 100;
```

Comment:
This struct is for describing the type(input/output type), not for serialization. TensorProto is the corresponding one for serialization. As you can see, Tensor and TensorProto are different.

Reply:
What I can see, TimePoint is more like the TensorProto, not the Tensor.

Reply:
Yes, it is for input/output

---
In reply to: [315352438](https://github.com/microsoft/onnxruntime/pull/1649#discussion_r315352438) [](ancestors = 315352438)

Reply:
Why can I not re-use the same definition say for tests input or expected output?

---
In reply to: [315352704](https://github.com/microsoft/onnxruntime/pull/1649#discussion_r315352704) [](ancestors = 315352704)

Reply:
The TypeProto  is for describing the type, not the value.  

BTW, is TimePoint  is an opaque type or not?

Reply:
Okay, thanks for clarification. Yes, currently I am masqerading under Opaque type but in the future such types would become on their own.

---
In reply to: [315357164](https://github.com/microsoft/onnxruntime/pull/1649#discussion_r315357164) [](ancestors = 315357164)